examples/webgl_animation_skinning_morph.html

```diff
@@ -254,6 +260,62 @@
 
 				gui.add( API, 'show skeleton' ).onChange( function() { helper.visible = API[ 'show skeleton' ]; } );
 
+
+				// Note: .add/removeAction and .enabled = true / false have
+				// different performance characteristics:
+				//
+				// The former changes dynamic data structures in the mixer,
+				// therefore the switch is more expensive but removes the
+				// per-action  base cost caused by the unique property
+				// bindings it uses.
+				//
+				// The latter is a zero-cost switch, but the per-frame base
+				// cost for having the action added to the mixer remains. 
+
+				function actionControls( key, action ) {
+
+					var guiNameAddRemove = key + ' action';
+					var guiNameEnabled = key + ' enable';
+
+					// set initial state
+
+					if ( API[ guiNameAddRemove ] ) {
+
+						action.enabled = API[ guiNameEnabled ];
+						mixer.addAction( action );
+
+					}
+
+					// attach controls
+
+					gui.add( API, guiNameAddRemove ).onChange( function() {
+
+						if ( API[ guiNameAddRemove ] ) {
```

Comment:
Probably having a temporary stupidity attach, but I can't figure it out. Please help!